apps/src/lib/ui/accounts/AdminAccountDialog.jsx

```diff
@@ -0,0 +1,68 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import i18n from '@cdo/locale';
+import color from '@cdo/apps/util/color';
+import BaseDialog from '@cdo/apps/templates/BaseDialog';
+import {Header, ConfirmCancelFooter} from '../SystemDialog/SystemDialog';
+
+const GUTTER = 20;
+
+export default class AdminAccountDialog extends React.Component {
```

Comment:
For new React components, we're supposed to use the functional style instead of classes. Example [here](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/lib/ui/LtiSectionSyncDialog.jsx)